tests/unit/containers/my-quizzes/CreateOrEditQuizContainer.spec.jsx

```diff
@@ -113,37 +112,24 @@ describe('CreateOrEditQuizContainer with id', () => {
   beforeEach(() => {
     useParams.mockReturnValue({ id: mockId })
 
-    getQuizSpy = vi.spyOn(ResourceService, 'getQuiz').mockResolvedValue({
-      data: {
+    mockAxiosClient
+      .onGet(new RegExp(URLs.quizzes.getById.replace(':id', mockId)))
```

Comment:
You need to put `RegExp` only if there are query params inside url
```suggestion
      .onGet(URLs.quizzes.getById.replace(':id', mockId))
```